eng/pipelines/templates/stages/archetype-python-release.yml

```diff
@@ -205,6 +205,16 @@ stages:
                 inputs:
                   versionSpec: '3.9'
 
+              - task: Powershell@2
+                inputs:
+                  filePath: /eng/common/scripts/Mark-ReleasePlanCompletion.ps1
+                  arguments: >
+                    -PackageInfoFilePath '${{parameters.ConfigFileDir}}/${{parameters.PackageName}}.json'
```

Comment:
Where does ConfigFileDir come from? I don't see it as a parameter.

Reply:
Fixed it. thanks